src/native/third_party/isa-l.gyp

```diff
@@ -129,7 +129,6 @@
                 'isa-l_crypto/include/types.h',
                 'isa-l_crypto/include/multi_buffer.h',
                 'isa-l_crypto/md5_mb/md5_ctx_base.c',
-                'isa-l_crypto/md5_mb/md5_ctx_base_aliases.c',
```

Comment:
Don't you need to remove aliases from sha1 too ?? or is it different ?

Reply:
Also can you add the bases of the rest of the hashes defined in this file?

Reply:
seems like sha1 have sha1_ctx_base_aliases.c in this version

Reply:
it is not - I guess I should remove?